portfolio/src/main/webapp/page.css

```diff
@@ -0,0 +1,136 @@
+:root {
+  --blue-default: #25274d;
+  --blue-secondary: #29648a;
+  --blue-active: #2E9cca;
+  --text-color: #fff;
+}
+
+/* 
```

Comment:
CSS styles go least to most specific. I think I may have gone dyslexic and told you the wrong thing originally. Sorry about that >_<